README.md

```diff
@@ -4,14 +4,14 @@
 The library of UI components used for Terminus applications.
 
 [![CircleCI][circle-badge]][circle-link]
-[![codecov][codecov-badge]][codecov-project]
+[![Codacy Coverage][codacy-coverage-badge]][codacy-project]
+[![Codacy Grade][codacy-grade-badge]][codacy-project]
 [![semantic-release][semantic-release-badge]][semantic-release]
 [![MIT License][license-image]][license-url]
 [![ZenHub][zenhub-image]][zenhub-url]
 <br>
 [![NPM version][npm-version-image]][npm-url]
 [![Github release][gh-release-badge]][gh-releases]
-[![Library size][file-size-badge]][raw-distribution-js]
 <br>
```

Comment:
Do we not want to show library size any more?

Reply:
I'd love to still show the library size, but this tool is looking for a single file to check the weight, but our library isn't packaged into a single file. So the tool was reporting a very incorrect weight since it just saw the top level `public-api` file.

At some point when I have free time, I'd love to add a simple gulp script that packaged it up as a single file just for the size reporting purpose.

Reply:
Safe to call this a negative of individual endpoints?

Reply:
More a result of how browsers are becoming able to support imports. So the need to put all our code in a single file isn't there anymore which is why our build tool dropped it.